app/screens/settings/advanced/index.tsx

```diff
@@ -32,26 +37,51 @@ const EMPTY_FILES: ReadDirItem[] = [];
 
 type AdvancedSettingsProps = {
     componentId: string;
-}
+};
 const AdvancedSettings = ({componentId}: AdvancedSettingsProps) => {
     const theme = useTheme();
     const intl = useIntl();
     const serverUrl = useServerUrl();
-    const [dataSize, setDataSize] = useState<number|undefined>(0);
+    const [dataSize, setDataSize] = useState<number | undefined>(0);
     const [files, setFiles] = useState<ReadDirItem[]>(EMPTY_FILES);
     const styles = getStyleSheet(theme);
 
     const getAllCachedFiles = async () => {
-        const {totalSize, files: cachedFiles} = await getAllFilesInCachesDirectory(serverUrl);
+        const {totalSize = 0, files: cachedFiles} = await getAllFilesInCachesDirectory(serverUrl);
         setDataSize(totalSize);
         setFiles(cachedFiles || EMPTY_FILES);
     };
 
     const onPressDeleteData = preventDoubleTap(async () => {
         try {
             if (files.length > 0) {
-                await deleteFileCache(serverUrl);
-                await getAllCachedFiles();
+                const {formatMessage} = intl;
+
+                Alert.alert(
+                    formatMessage({
+                        id: t('advanced_settings.delete_data'),
+                        defaultMessage: 'Delete Documents & Data',
+                    }),
+                    formatMessage({
+                        id: t('mobile.advanced_settings.delete_message.confirmation'),
+                        defaultMessage: '\nThis will delete all offline data downloaded through the app. Please confirm to proceed.\n',
```

Comment:
This will delete all offline files and cached data downloaded through the app.

Reply:
Does cache data means the database content? Cause removing the database content means that it will need to do a "cold start" as in calling appEntry.. thus I think we can keep "cache data" out of it

Reply:
Ah, so what is actually happening when this button is pressed? What would you propose we call it?

Reply:
What happens is that we are deleting the caches folder for both platform.  The type of data contained in there are images, videos or documents.

Reply:
Thanks. 0/5 I'll leave it to you guys to pick the text, it could be fine as is.

Reply:
I propose " This will delete all files downloaded through the app for this server. Please confirm to proceed. "

Reply:
Go for it, we can change it later if needed

Reply:
corrected